imgui-binding/src/main/java/imgui/extension/imguizmo/ImGuizmo.java

```diff
@@ -0,0 +1,240 @@
+package imgui.extension.imguizmo;
+
+import imgui.ImDrawList;
+import imgui.ImGui;
+import imgui.ImVec2;
+
+
+import java.util.List;
+
+public class ImGuizmo {
+    private static final boolean[] buffer = new boolean[1];
+    private static final float[] matrixBuffer = new float[16];
+    private static final float[] positionBuffer = new float[3];
+    private static final float[] rotationBuffer = new float[3];
+    private static final float[] scaleBuffer = new float[3];
+
+    /*JNI
+       #include <imgui.h>
+       #include <ImGuizmo.h>
+       #include "jni_common.h"
+    */
+    private native static void nEnabled(boolean[] buffer); /*
+        ImGuizmo::Enable(&buffer[0]);
+    */
+
+    /**
+     * This will set the orthographic view
+     */
+    public static void setEnabled(boolean isEnabled) {
+        buffer[0] = isEnabled;
+        nEnabled(buffer);
+    }
+
+    private native static void nIsUsing(boolean[] buffer); /*
+        buffer[0] = ImGuizmo::IsUsing();
+    */
+
+    /**
+     * Natively checks to see if we're using the guizmo
+     */
+    public static boolean isUsing() {
+        nIsUsing(buffer);
+        return buffer[0];
+    }
+
+    private native static void nIsOver(boolean[] buffer); /*
+        buffer[0] = ImGuizmo::IsOver();
+    */
+
+    /**
+     * Natively checks to see if we're over the guizmo
+     */
+    public static boolean isOver() {
+        nIsOver(buffer);
+        return buffer[0];
+    }
+
+    private native static void nSetDrawList(long pointer); /*
+        ImGuizmo::SetDrawlist((ImDrawList*)pointer);
+    */
+
+    /**
+     * This will set the draw list of the given guizmo
+     */
+    public static void setDrawList(ImDrawList drawList) {
+        nSetDrawList(drawList.ptr);
+    }
+
+
+    /**
+     * This will set the default window drawlist value
+     */
+    public static void setDrawList() {
+        setDrawList(ImGui.getWindowDrawList());
+    }
+
+    private native static void nBeginFrame(); /*
```

Comment:
You don't need this method. Just make `public static native void beginFrame` and call native function there. So it should be like `call native method` not `call java method->which calls native method`. Java layer is needed to do custom logic, like extraction of internal data from types `ImFloat` etc. Other than that, do direct native calls.